### PR TITLE
Added check if user account is verified or not

### PR DESCRIPTION
**Pull request by adrean-hipe**
OVERVIEW

- Added check in LoginController if the user is_verified
- Added error message in login view if the user tries to login with unverified account

NOTES

- Only verified accounts are able to login and proceed to home page

### CODE REVIEW

File: laravel-app/app/Http/Controllers/Auth/LoginController.php [hunk 1]
```diff
@@ -37,4 +39,13 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function authenticated(Request $request, $user)
```
Comment (hipe01mark) [inline laravel-app/app/Http/Controllers/Auth/LoginController.php:43]: Dont expose your validation request in your controller. Use **Form Request** instead.

Comment (bpocrafael) [reply laravel-app/app/Http/Controllers/Auth/LoginController.php:43]: also you are breaking the principle of Single responsibility in SOLID Principle

Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/Auth/LoginController.php]: Changed to a Form Request instead sir. Thanks ❤️

Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/Auth/LoginController.php:25]: Separated the files that does login and verifying the user. Thanks ❤️

File: laravel-app/app/Http/Controllers/Auth/LoginController.php [hunk 2]
```diff
@@ -33,8 +37,15 @@ class LoginController extends Controller
...
+        $this->userVerificationService = $userVerificationService;
     }
-}
+
+    protected function authenticated(Request $request, $user)
+    {
+        $this->userVerificationService->isUserVerified($user);
+    }
+
+}
```
Comment (bpocrafael) [inline laravel-app/app/Http/Controllers/Auth/LoginController.php:51]: add new line

File: laravel-app/app/Services/UserVerificationService.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+namespace App\Services;
+
+use Illuminate\Validation\ValidationException;
+
+class UserVerificationService
+{
+    public function isUserVerified($user)
+    {
+        if (!$user->is_verified) {
+            auth()->logout(); // Log the user out if not verified
```
Comment (bpocrafael) [inline laravel-app/app/Services/UserVerificationService.php:12]: nit, remove comments

File: laravel-app/app/Http/Controllers/Auth/LoginController.php (hunk 2 above)
Comment (adrean-hipe) [reply laravel-app/app/Http/Controllers/Auth/LoginController.php]: Commited sir. Thanks ❤️

File: laravel-app/app/Services/UserVerificationService.php (hunk 3 above)
Comment (adrean-hipe) [reply laravel-app/app/Services/UserVerificationService.php]: Commited sir. Thanks ❤️

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Review by bpocrafael — APPROVED** [review]
LGTM 🐑

**Pull request merged**